### PR TITLE
Add wrapper

### PR DESCRIPTION
**Pull request by alexvpickering**
Adds a wrapper to ingest multiple experiments at the same time. Also exposes `CLUSTER_ENV=all` to upload to both production and staging.

This will probably be a stop gap until data-ingest is moved to the pipeline.

### CODE REVIEW

File: src/wrapper/ingest-all.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ rm -rf input
+ cp -R $d/. input
+ docker-compose up --build
+
+ # add to exp_meta.csv file
+ Rscript ./src/wrapper/add_exp_meta.R
+
+ # can't remove output for next as from docker root user
+ # can rename
+ rename output output_$f output
```
Comment (JuanluOnieva-biomage) [inline src/wrapper/ingest-all.sh:16]: could this increase the size of the docker container?

I think it's better to dump the content out of the data-ingest, as I don't know if leaving it in the data-ingest will make the docker image too heavy.

**Review by JuanluOnieva-biomage — COMMENTED** [review]
Very good work Alex! Thanks to your code we will be able to reflect the changes faster. After solving my doubt, I'm happy to merge!

File: src/wrapper/ingest-all.sh (hunk 1 above)
Comment (alexvpickering) [reply src/wrapper/ingest-all.sh:16]: I think you're right. The biggest adding is going to be the accumulating `output_*` directories which can't be removed or moved (unless we change the docker user to the current user - didn't look portable to me). I think the easiest workaround is I'll just add a `.dockerignore` with `output_*` and `user_data/*`.

Comment (JuanluOnieva-biomage) [reply src/wrapper/ingest-all.sh:16]: Happy with the solution! Do you mind to create and push `.dockerignore` file?

**Review by JuanluOnieva-biomage — APPROVED** [review]

**Pull request merged**